src/views/teams/TeamList.jsx

```diff
@@ -26,7 +26,7 @@ function TeamList(){
                 await deleteTeamById(id)
                 await loadTeams()
             } catch (error) {
-                alert(error.message)
+                alert("Must delete all of the team's players before deleting the team.")
```

Comment:
Hmm, now it appears that if I do delete the player on a team, the player will still show on the team details page and won't allow me to delete the team.